pallets/gear/src/manager/task.rs

```diff
@@ -170,6 +170,13 @@ where
             message_id,
             MessageWokenRuntimeReason::WakeCalled.into_reason(),
         ) {
+            Pallet::<T>::deposit_event(Event::MessageEnqueued {
```

Comment:
we mustn't deposit here any event!
to match this test just check queue::contains()

Reply:
> `QueueOf` doesn't have method `contains`, and even we query it from `DispatchesWrap` this message doesn't exist...
> 
> this task processes right atfer we quening it, and there is no event showing it gets queued ( without providing this event ) or processed ( we have MessageDispatched but no message_id to verify tbh ) =,=

Then provide logic for demo that this message will trigger user message sent